INSTALLATION.md

```diff
@@ -9,9 +9,9 @@ check the [TL;DR section](#tldr).
 
 ## Installing Atomkraft
 
-Installing Atomkraft requires [Python](https://www.python.org) 3.10, `git` and [`java` 17 or newer](https://apalache.informal.systems/docs/apalache/installation/jvm.html).
+Installing Atomkraft requires [Python](https://www.python.org) 3.8 or newer (we suggest using Python 3.10), together with its package manager [pip](https://pip.pypa.io/en/stable/installation/).
 
-If you don't have Python 3.10 already installed, please consult the [installation instructions](https://realpython.com/installing-python/) for different installation options.
+If you don't have Python already installed, please consult the [installation instructions](https://realpython.com/installing-python/) for different installation options.
```

Comment:
Please remove `realpython` tutorial. It's not official and outdated.

Reply:
We can share something like this - https://wiki.python.org/moin/BeginnersGuide/Download

I am in awe that Python doesn't have any unified installation guide.